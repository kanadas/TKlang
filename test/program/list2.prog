
concat l1 = (\l2 -> match l1 with {
    h:t -> h : concat t;
    [] -> l2}) [1, 2, 3, 4] 
