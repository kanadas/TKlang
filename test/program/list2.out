program/list2.prog
No main stream. Nothing to do...
